src/main/java/reposense/RepoSense.java

```diff
@@ -62,7 +67,10 @@ public static void main(String[] args) {
                     cliArguments.getSinceDate().orElse(null),
                     cliArguments.getUntilDate().orElse(null));
 
-            FileUtil.zip(cliArguments.getOutputFilePath().toAbsolutePath(), ".json");
+            HashSet<Path> relevantPaths = getRelativeFolders(cliArguments.getOutputFilePath(), configs);
+            relevantPaths.add(Paths.get(cliArguments.getOutputFilePath().toAbsolutePath().toString(),
+                    SummaryReportJson.SUMMARY_JSON_FILE_NAME));
+            FileUtil.zipRelativeFiles(relevantPaths, cliArguments.getOutputFilePath().toAbsolutePath(), ".json");
```

Comment:
Word choice suggestion. Try report folders/files?

You use both relative and relevant. Report folder/file is more consistent and it implies that they are relevant.

You may be thinking of naming this a more general method 🤔 but I think zipping is only used for report archival.

Reply:
I renamed: -

- the method `getRelativeFolders` to `getReportFolders`,
- the method `zipRelativeFiles` to `zipReportFoldersAndFiles`,
- the variables `relevantPaths` and `repoFoldersAndFiles`.

Would these names be better?

Reply:
No need to embed source code, this should save you some time. 😃


>zipRelativeFiles to zipRelevantFoldersAndFiles

```public static void zipRelevantFoldersAndFiles(HashSet<Path> relevantPaths,```

If you specify paths (**with intention**), those are already relevant right?
Do developers need to be aware of *relevant* or *irrelevant* paths?

```public static void zipFoldersAndFiles(HashSet<Path> pathsToZip,``` may remove some confusion.

Reply:
Ok noted